src/test/java/com/epam/tc/hw2/BasePageTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public class BasePageTest {
+
+    protected WebDriver driver;
+    protected WebDriverWait driverWait;
+    private static final long TIMEOUT_DURATION = 10;
+
+    @BeforeSuite
+    protected void setUpDriver() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
```

Comment:
Here should be BeforeClass, I recon.

Reply:
I wanted the page to be closed after each test method (it may be more than one in the class), so I left BeforeMethod